platforms/web/test-e2e/tests/payments/payments_test.ts

```diff
@@ -31,7 +31,7 @@ const cleengProps: ProviderProps = {
   cardInfo: Array.of(['Card number', '•••• •••• •••• 1115'], ['Expiry date', '03/2030'], ['Security code', '******']),
 };
 
-runTestSuite(jwProps, 'JW Player');
+// runTestSuite(jwProps, 'JW Player');
```

Comment:
This is not finished yet? Will it be possible to do e2e testing with the access bridge? 🤔